CMakeLists.txt

```diff
@@ -54,6 +54,15 @@ include_directories("${CMAKE_CURRENT_BINARY_DIR}" "./src" "./third_party/grpc" "
 #----------------------------------------------------------------------
 # LabVIEW support for grpc and protobuf
 #----------------------------------------------------------------------
+
+add_custom_target(Detect_MessageMetadata_Structural_Changes
+    COMMAND ${CMAKE_COMMAND} -E echo "Running MessageMetadata structure test.py ..."
+    COMMAND python -m pip install -r ${CMAKE_CURRENT_SOURCE_DIR}/tests/CMakeTests/requirements.txt
```

Comment:
instead of having requirements.txt, create a python script to create venv and run the compatibility checker inside that venv